pkg/securityscan/job/job.go

```diff
@@ -68,6 +74,12 @@ func New(clusterscan *cisoperatorapiv1.ClusterScan, clusterscanprofile *cisopera
 						cisoperatorapi.LabelProfile:     clusterscan.Spec.ScanProfileName,
 						cisoperatorapi.LabelClusterScan: clusterscan.Name,
 					},
+					OwnerReferences: []metav1.OwnerReference{{
+						APIVersion: "cis.cattle.io/v1",
+						Kind:       "ClusterScan",
+						Name:       clusterscan.Name,
+						UID:        clusterscan.GetUID(),
+					}},
```

Comment:
i don't think this is necessary. The job owns the pod it creates

Reply:
updated